src/main/java/uk/gov/justice/hmpps/offenderevents/subscribe/PrisonerEventsListener.java

```diff
@@ -39,8 +39,8 @@ public PrisonerEventsListener(ObjectMapper objectMapper,
         this.delay = delay;
     }
 
-    @JmsListener(destination = "prisoneventqueue", containerFactory = "hmppsQueueContainerFactoryProxy")
-    public void onPrisonerEvent(String message, SQSTextMessage rawMessage) throws JsonProcessingException {
+    @SqsListener(queueNames = "prisoneventqueue", factory = "hmppsQueueContainerFactoryProxy", maxInflightMessagesPerQueue = "1", maxMessagesPerPoll = "1")
```

Comment:
I'm not sure this 1 message at a time thing works - offender-events-ui kept rejecting messages due to no threads.